include/HUDL/HUDL.hpp

```diff
@@ -105,18 +102,24 @@ class HUDL {
     DEV::LCD lcd;
     uint16_t voltages[1] = {};
 
-    uint32_t thermTemps[5] = {0, 0, 0, 0};
+    uint32_t thermTemps[4] = {};
 
     static constexpr uint16_t OBJECT_DICTIONARY_SIZE = 30;
+    static constexpr uintptr_t TMS_TPDO0_COB_ID = CO_COBID_TPDO_DEFAULT(0);
```

Comment:
@mjh9585 should correct me if I'm wrong, but I think he meant that to get the message to receive correctly, you should use `.Data = (uintptr_t) CO_COBID_TPDO_DEFAULT(0) + SENDER_NODE_ID,` where `SENDER_NODE_ID` is the constant you're storing, so you can use it for multiple RPDOs. I'm not sure how you were able to get the RPDOs to work without setting the COBID of the TPDO correctly, so I am worried this is going to break what you were doing before.

Reply:
Ok I better understand what you guys are saying now. So the reason it works with the TMS then is because in the TMS code, the node ID is not added and its just TPDO_DEFAULT(0) and TPDO_DEFAULT(1)

Reply:
I can store the sender IDs here but it will most likely not work with the TMS until that gets fixed there

Reply:
Yeah, for now you can test with the sender ID of 0 until that gets fixed. It looks like it will be fixed as soon as this pull request is approved: https://github.com/RIT-EVT/TMS/pull/11